authors/apps/friends/views.py

```diff
@@ -34,17 +44,23 @@ def post(self, request, username, format=None):
                 {"message": "You cannot follow yourself"},
                 status=status.HTTP_406_NOT_ACCEPTABLE)
 
-        if Friend.objects.filter(user_from=follower, user_to=followed):
+        is_following = Friend.objects.get_or_create(
+            user_from=follower, user_to=followed)
+        if is_following[1] is False:
             return Response(
                 {"message": "you already follow this user"},
                 status=status.HTTP_406_NOT_ACCEPTABLE)
 
-        Friend.objects.get_or_create(user_from=follower, user_to=followed)
-        user = get_user_model().objects.get(pk=followed.id)
-        serializer = FollowUnfollowSerializer(user)
-        user_id = serializer.data
-        if serializer:
-            return Response(user_id, status=status.HTTP_200_OK)
+        number_of_followers = self.get_number_of_followers(followed)
```

Comment:
I don't think we need to include these in the response message. It feels like unnecessary db hits. I'm fine with the previous implementation asides the comment i made.